### ISSUE TITLE
Support more social icons

### ISSUE THREAD

**Issue by nishanths** [nishanths/cocoa-hugo-theme#50]
Currently, we only support social icons for twitter, github, linkedin, email, and RSS.

We could incrementally add support for:

- [ ] telegram
- [ ] pinterest
- [ ] keybase
- [ ] foursquare
- [ ] are.na

and so on.

These can be enabled/disabled in `config.toml` as usual.

**Comment by mtoto** [comment]
+1 stackoverflow would be nice too

**Comment by elonzh** [comment]
Why not use [`fontawesome`](http://fontawesome.io/icons/) ?

Simply using params in config and render them in template.

**Comment by nishanths** [comment]
Yes, fontawesome is a good option. It's a large library, hence the hesitation to include it. Ideally, we have individual SVGs for just the icons we need.

Happy to include fontawesome behind a `config.toml` flag.
